src/config.nim

```diff
@@ -27,9 +27,13 @@ type
     vhosts*: Table[string, VHost]
     homeDir*: string
     dirHeader*: string
+    titanSettings*: TitanSettings
+
+  TitanSettings* = object
```

Comment:
The fields dont need the `titan` prefix anymore.
Same applies to the params in the config file